src/api/app/models/bs_request.rb

```diff
@@ -52,6 +52,8 @@ class SaveError < APIException
   has_many :request_history_elements, -> { order(:created_at) }, class_name: 'HistoryElement::Request', foreign_key: :op_object_id
   has_many :review_history_elements, through: :reviews, source: :history_elements
 
+  belongs_to :user, foreign_key: 'creator', primary_key: 'login'
```

Comment:
But with this change we could / should store the user id, shouldn't we? And I don't see a stored login in the bs_requests table.